acs-integration-tests/pom.xml

```diff
@@ -90,6 +90,9 @@
                 <artifactId>maven-failsafe-plugin</artifactId>
                 <version>2.18.1</version>
                 <configuration>
+                    <forkCount>2.0C</forkCount>
```

Comment:
This is a good addition however you have to remove redefinitions of `forkCount` and `parallel` properties from all other places in this file otherwise no Failsafe-level parallelization will be done :)